app/[domain]/[lang]/[plan]/(with-layout-elements)/actions/[id]/page.tsx

```diff
@@ -55,6 +65,7 @@ export async function generateMetadata(
       description: data.action.name,
       images: image?.social?.src ? [image.social.src] : undefined,
       url: resolvedParent.openGraph?.url ?? undefined,
+      siteName,
```

Comment:
Nicely done, but we can simplify this by using the parent route's metadata. `parent` is automatically passed to to `generateMetadata` as a promise which includes all parent metadata. See `url` above for an example of this.

Reply:
Cool :) Thanks, done